### PR TITLE
report non-skewed uptime from domain-server

### PR DESCRIPTION
**Pull request by birarda**
- calculate uptime on domain-server instead of client javascript to remove effects of clock skew

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2738f65c10b624a7e49d3edc7d935b8df1ba8d96 available at: https://gist.github.com/737b68cdc3db0d96225c<br/><br/>No installer was built for this commit

**Comment by hifi-gustavo** [comment]
Build console output for commit: b386a95d0852a4284d414e26ac6bbd56d95fed44 available at: https://gist.github.com/737b68cdc3db0d96225c<br/><br/>No installer was built for this commit

File: domain-server/src/DomainServer.cpp [hunk 1]
```diff
@@ -1502,7 +1502,7 @@ QJsonObject DomainServer::jsonObjectForNode(const SharedNodePointer& node) {
     nodeJson[JSON_KEY_LOCAL_SOCKET] = jsonForSocket(node->getLocalSocket());
 
     // add the node uptime in our list
-    nodeJson[JSON_KEY_WAKE_TIMESTAMP] = QString::number(node->getWakeTimestamp());
+    nodeJson[JSON_KEY_UPTIME] = QString::number((QDateTime::currentMSecsSinceEpoch() - node->getWakeTimestamp()) / 1000.0);
```
Comment (AndrewMeadows) [inline domain-server/src/DomainServer.cpp]: There is an implicit cast here for the numerator: **quint64** to **double**.

**Comment by hifi-gustavo** [comment]
Build console output for commit: aac15e52bb223fac707f9c2f4e77ab6982ef11d1 available at: https://gist.github.com/737b68cdc3db0d96225c<br/><br/>No installer was built for this commit

**Pull request merged**